examples/MvcWeb/TheraLang/Configuration/ProjectConfiguration.cs

```diff
@@ -16,7 +16,13 @@ public void Configure(EntityTypeBuilder<Project> builder)
             builder.Property(x => x.Name).HasMaxLength(250).IsRequired();
             builder.Property(x => x.Type).HasMaxLength(250).IsRequired();
 
-            builder.Property(e => e.StatusId).HasMaxLength(250).IsRequired(); //TODO: HasDefaultValue
+            builder.HasMany(x => x.ResourceProjects).WithOne(i => i.Project).HasForeignKey("ProjectId");
+            builder.HasMany(x => x.ProjectParticipations).WithOne(i => i.Project).HasForeignKey("ProjectId");
+            builder.Property(e => e.Name).HasMaxLength(250).IsRequired();
+
+            builder.Property(e => e.Type).HasMaxLength(250).IsRequired();
+
+            builder.Property(e => e.StatusId).HasMaxLength(250).IsRequired().HasDefaultValue(0); //TODO: HasDefaultValue
```

Comment:
Why you haven't added default value from enum?
builder.Property(e => e.StatusId).HasMaxLength(250).IsRequired().HasDefaultValue(Entities.ProjectStatus.New);